tests/Services/pdg.config.yaml

```diff
@@ -6,7 +6,7 @@ pdg:
       directories: [ 'Controller', 'Validator' ]
       names: [ '*.php' ]
       exclude: [ '*Interface.php' ]
-      class_tags_to_ignore: [ '@internal', '@experimental' ]
+      tags_to_ignore: [ '@internal', '@experimental', '@ignore' ]
```

Comment:
Invalid file according to new configuration: `patterns` node has been removed, and some other options too

Reply:
Anyway: as ConfigurationHandler class has been removed, this file should be moved in favor of ConfigurationTest